sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleExportDto.java

```diff
@@ -279,7 +291,7 @@ public void setRequestedOtherPathogenTests(String requestedOtherPathogenTests) {
 		this.requestedOtherPathogenTests = requestedOtherPathogenTests;
 	}
 
-	@Order(20)
+	@Order(21)
```

Comment:
FYI: There was no need to increment this and the following order entries. We have left some unused order entries (e.g. 18 and 19) to allow for easy addition of new fields without having to update all following order entries.
You can leave it like that now.

Reply:
Alright